### ISSUE TITLE
disable auto search from drop downs

### ISSUE THREAD

**Issue by ManjotS** [KhalisFoundation/sttm-web#982]
It  is annoying that it auto  searches when drop downs are changed. For example, if  I am  on an ang and want to search a first letter or english (informed by what ang I am  looking at) I can no longer write the query in the correct font mode.

Search  should be done when theres actually a search button  click.

**Comment by gurjit03** [comment]
@saintsoldierx do we require this feature?

**Comment by saintsoldierx** [comment]
Yes, we do need to look at this logic and iron out the quirks @gurjit03

**Comment by gurjit03** [comment]
@saintsoldierx I will check it out
